source/jobs/README.md

```diff
@@ -0,0 +1,232 @@
+# Jobs
+ **Notice:** Running the AWS IoT Device Client will incur usage of AWS IoT services, and is likely to incur charges on your AWS account. Please refer the pricing pages for [AWS IoT Core](https://aws.amazon.com/iot-core/pricing/), [AWS IoT Device Management](https://aws.amazon.com/iot-device-management/pricing/), and [AWS IoT Device Defender](https://aws.amazon.com/iot-device-defender/pricing/) for more details.
+
+  * [Jobs Feature](#jobs-feature)
+    + [Creating a Job](#creating-a-job)
+    + [Creating your own Job Handler](#creating-your-own-job-handler)
+      - [Job/Job Handler Security Considerations](#jobjob-handler-security-considerations)
+    + [Debugging your Job](#debugging-your-job)
+    + [Jobs Build Flags](#jobs-build-flags)
+    + [Jobs Feature Configuration Options](#jobs-feature-configuration-options)
+      - [Configuring the Jobs feature via the command line](#configuring-the-jobs-feature-via-the-command-line)
+      - [Configuring the Jobs feature via the JSON configuration file](#configuring-the-jobs-feature-via-the-json-configuration-file)
+
+[*Back To The Main Readme*](../../README.md)
+
+## Jobs Feature
+
+The Jobs feature within the AWS IoT Device Client provides device-side functionality for execution of jobs created via
+https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html. When the Jobs feature starts within the AWS IoT Device Client, 
```

Comment:
nitpick: Same as above, let's replace this hyperlink instead of displaying the full link.